pkg/cassandra/datacenter.go

```diff
@@ -331,6 +333,10 @@ func Coalesce(clusterName string, clusterTemplate *api.CassandraClusterTemplate,
 		dcConfig.MgmtAPIHeap = dcTemplate.DatacenterOptions.MgmtAPIHeap
 	}
 
+	if dcTemplate.CDC != nil {
+		dcConfig.CDC = dcTemplate.CDC
+	}
```

Comment:
```suggestion
	if dcTemplate.CDC == nil {
		dcConfig.CDC = clusterTemplate.CDC
	} else {
	        dcConfig.CDC = dcTemplate.CDC
	}
```

Reply:
No, no. I intended exactly what I'd written. We don't allow definition of CDC feeds cluster-wide, they are defined per DC. Think about where you want the data to flow (and how many copies you want to land in each Pulsar cluster) and you'll understand why.